weed/wdclient/vid_map.go

```diff
@@ -177,6 +190,10 @@ func (vc *vidMap) addEcLocation(vid uint32, location Location) {
 }
 
 func (vc *vidMap) deleteLocation(vid uint32, location Location) {
+	if vc.cache != nil {
+		vc.cache.deleteLocation(vid, location)
+	}
+
```

Comment:
The delete operation will be performed on all cache nodes

Reply:
If addition is also applied to cache, we just need one layer of cache.